test/test_ops.py

```diff
@@ -771,20 +777,37 @@ def test_variant_consistency_jit(self, device, dtype, op):
         include_conjugated_inputs = op.test_conjugated_samples and dtype.is_complex
         samples = op.sample_inputs(device, dtype, requires_grad=_requires_grad, include_conjugated_inputs=include_conjugated_inputs)
 
-        for sample in samples:
-            # Acquires variants to test
-            func = op.get_op()
-            method = op.get_method()
-            variants = {
-                # TODO: inplace tests currently fail, fix and add inplace variant
-                'function': func, 'method': method,
-            }
+        # Acquires variants to test
+        func = op.get_op()
+        method = op.get_method()
+        variants = {
+            # TODO: inplace tests currently fail, fix and add inplace variant
+            'function': func, 'method': method,
+        }
+
+        # TODO: find better way to standardize on op registration itself..
+        has_fake_function = op.name in ["resize_", 'resize_as_']
+
+        if has_fake_function:
+            variants = {'method': getattr(torch.Tensor, op.name)}
+            samples = op.sample_inputs(device, dtype, requires_grad=False)
 
+        tested = False
+        for sample in samples:
             # Test traced and scripted consistency
             for func_type, variant in variants.items():
                 if variant is None:
                     continue
 
+                # scripting and check_alias_analysis do not work with lambdas
+                # lambdas are typically used as a way to simulate methods without
+                # functional variants, so rely on the other variant for testing
+                # for now
+                if is_lambda(variant):
```

Comment:
Would this be replaced by querying the "has_fake_function" attribute?

Reply:
is `has_fake_function` exactly the same thing as lambda ? I think the fake function is where we are adding extra nodes as a way to not mutate the inputs. There are other nodes with a lambda like torch.where

Reply:
It can be whatever we want. I'd design it so it's most useful to you so long as it's clear to users how to set the metadata